Documentation/coding-guidelines/clr-jit-coding-conventions.md

```diff
@@ -16,6 +16,8 @@ Note that these conventions are different from the CLR C++ Coding Conventions, d
 
 > Note: the JIT currently doesn't follow some of these conventions very widely. The non-conformant code should be updated, eventually.
 
+> Node: we now use jit-format to format our code. All changes it makes supersedes the conventions in this doc. Please see the [https://github.com/dotnet/jitutils/blob/master/doc/getstarted.md](jit-format documentation) for instructions on running jit-format.
```

Comment:
It looks like you have `[URL](link text)` here.  It should be `[link text](URL)`